packages/jest-types/src/Global.ts

```diff
@@ -17,7 +22,7 @@ export type ConcurrentTestFn = (
 ) => Promise<void | undefined | unknown>;
 export type BlockFn = () => void;
 export type BlockName = string;
-export type HookFn = TestFn;
+export type HookFn = TestFn | GeneratorFn;
```

Comment:
test functions can also be generators, so change that definition instead?

Reply:
Thanks, I couldn't be sure just by reading the docs. On it.